.github/workflows/main.yml

```diff
@@ -70,6 +70,38 @@ jobs:
           file: build/*/*.dmg
           tag: ${{ github.ref }}
           file_glob: true
+          
+  appcast:
+    name: Set Appcast
+    needs: build
+    runs-on: macos-latest
+    env:
+      JOB_TYPE: BUILD
+      SIGNATURE_FOR_SIGNING: ${{ secrets.SIGNATURE_FOR_SIGNING }}
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Get Sparkle 1.26
+        run: |
+          curl -L -s "https://github.com/sparkle-project/Sparkle/releases/download/1.26.0/Sparkle-1.26.0.tar.xz" -o Sparkle.tar.xz || exit 1
+          tar -xf Sparkle.tar.xz
```

Comment:
Needs an `|| exit 1`?

Reply:
I thought it probably is not needed as extracting only works if the file is downloaded successfully. But thinking about it, an error check is probably a good idea.